lib/sycamore/sycamore/transforms/split_elements.py

```diff
@@ -34,12 +34,17 @@ def __init__(self, child: Node, tokenizer: Tokenizer, maximum: int, **kwargs):
     def split_doc(parent: Document, tokenizer: Tokenizer, max: int) -> Document:
         result = []
         for elem in parent.elements:
+            # Ensure the _header does not take up more than a third of the tokens
+            # Also avoid max resursive depth error
+            if elem.get("_header") and len(tokenizer.tokenize(elem["_header"])) / max > 0.33:
+                del elem["_header"]
             result.extend(SplitElements.split_one(elem, tokenizer, max))
         parent.elements = result
         return parent
 
     @staticmethod
     def split_one(elem: Element, tokenizer: Tokenizer, max: int) -> list[Element]:
+
         txt = elem.text_representation
         if not txt:
             return [elem]
```

Comment:
If the element has a `_header` but no `text_representation`, its `_header` won't be added in this code path. Is this a possible scenario?